lib/authority/controller.rb

```diff
@@ -144,6 +148,15 @@ def authority_forbidden(error)
       render :file => Rails.root.join('public', '403.html'), :status => 403, :layout => false
     end
 
+    # Log successful authorization attempt if enabled in the authority configuration.
+    #
+    # This method can be overloaded inside the application controller, similar to authority_forbidden.
+    def authority_success(user, action, resource)
+      if Authority.configuration.log_success
+        Authority.logger.info("#{user} successfully performed the #{action} action to this resource: #{resource}")
```

Comment:
One other thought - if `authority_success` can be overridden to do anything the user wants, the config option `log_success` isn't necessarily an accurate name. What if we don't even have a config option, and `authority_success` has a default implementation that does nothing? Then the instructions are "if you want to log successful authorization or do anything else about it, define `authority_success` however you like."